src/components/Image/ImageLoader.jsx

```diff
@@ -26,7 +25,7 @@ class ImageLoader extends React.Component {
     const { client } = this.props
     this._mounted = true
     this.loadNextSrc()
-    this.realtime = new Realtime({ client })
+    this.realtime = client.plugins.realtime.realtime
```

Comment:
We should use directly the instance of the plugin.

`client.plugins.realtime`

The instance of the plugin proxies to the CozyRealtime instance.
See https://github.com/cozy/cozy-notes/commit/6f0f92d833e014d06f125ba52ff3a7fc0bacac33#r38037516 for more context.

This is especially important in situations where we login/logout and keep the same client since we have to re-instantiate the CozyRealtime instance. THis is not done yet in the plugin but should be.

When CozyRealtime is reinstantiated, we'll have a new instance in `realtime.realtime`, but we'll keep a reference to the old instance here.